src/plugins/intel_gpu/include/intel_gpu/graph/serialization/activation_serializer.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
nit: copyright notices for the new source code
```suggestion
// Copyright (C) 2023 Intel Corporation
```

Reply:
I updated the copyright year. Thank you.